exercises/CI/introduction_to_ci/README.md

```diff
@@ -0,0 +1,154 @@
+  +     mjCuSVmrE4GrXhuIUT9mIJHf6s4TbRPxoR7X5aRDHRfmJda9GQiT4Wi8j14a8A/g
+  +     tWUtQOHxuHKYn+omreJE7MhIJH9l2iKFV3l1WsDb0VDtWO6ycIBH1BoJXIV+odtY
+  +     tMR/cQKBgE2L/lEzFR+/d80d39ECGGaCllC2F2cGz/BSWNle5Cp7PVs8y23yG3HG
+  +     iOCmPStUUZ7hi8OVjfsszJqv/z/Tp6kx/hNHXoPFmGcD4G6JyiifwlGG4tLMsO+g
+  +     SJm2WgmccOTu2RVpUkBcPKgxENQWO85QnTF9MlVjjuGRelWcrXwi
+  +     -----END RSA PRIVATE KEY-----
+  +   uri: ssh://git@git-server/git-server/repos/application-repo.git
+  
+jobs:
+  job my-first-job has been added:
+  + name: my-first-job
+  + plan:
+  + - do:
+  +   - aggregate:
+  +     - get: repo
+  +       trigger: true
+  +   - task: pointless-echo
+  +     config:
+  +       platform: linux
+  +       image_resource:
+  +         type: docker-image
+  +         source:
+  +           repository: alpine
+  +       run:
+  +         path: sh
+  +         args:
+  +         - -e
+  +         - -c
+  +         - -x
+  +         - |
+  +           echo "Something worked"
+  +       inputs:
+  +       - name: repo
+  
+pipeline created!
+you can view your pipeline here: 127.0.0.1:8080/teams/main/pipelines/ci-intro-pipeline
+
+the pipeline is currently paused. to unpause, either:
+  - run the unpause-pipeline command
+  - click play next to the pipeline in the web ui
+```
+
+
+
+Visit the [concourse home page](http://127.0.0.1:8080) again and you'll see the pipeline that the was just pushed.
+![Pending Pipeline](./pending_pipeline_screenshot.png)
+
+
+You'll notice the following:
+- there is now a pipe line called: `ci-intro-pipeline`
+- the pipeline is paused
+
+When pipelines are first pushed, they are not enabled. In order to run the pipeline we need to unpause it. Do this by either clicking the play symbol or by running: `./resources/linux/fly -t local unpause-pipeline -p ci-intro-pipeline`. Unpausing the pipeline should cause Concourse to trigger a build for our new pipeline. Go back to the [detailed view of the pipeline](127.0.0.1:8080/teams/main/pipelines/ci-intro-pipeline) after 20 seconds or so, you'll notice that that pipeline goes green. This means that the code has been pulled successfully from GIT and that the singe job, that is in the pipeline, has executed successfully.
+
+![Passing Pipeline](./passing_pipeline_screenshot.png)
+### Now it's your turn
+You are part of a team who has been working on a new application, a team member has carefully made a change to the code and is recommending that the change be released straight to production. Nothing could possibly go wrong!... could it?
+
+to complete this exercise do the following:
+
+1. Push the project pipeline to concourse - The pipeline file can be found in `./resources/application-pipeline.yml`
+**Note:** Make sure that it is enabled and runs successfully.
+2. The proposed change sits uncommitted in `./resources/checkout`. Commit and push your team mate's changes to GIT
```

Comment:
I cannot find the resources/checkout folder/file here. 
Also , it is not clear to me here what git magic was applied to make this work. We haven't mentioned in the readme that we're even spinning up a git instance for the purposes of this exervise, nor that we've pre-seeded it and pre-configured the command line to use it.

Reply:
This happened because the checkout in the after script failed. It'll be there when we get that fixed